symphony/app/fbcnms-projects/workflows/src/HttpServerSide.js

```diff
@@ -11,8 +11,8 @@
 import request from 'superagent';
 
 const HttpClient = {
-  get: (path, parentRequest) =>
-    new Promise((resolve, reject) => {
+  get: (path: string, parentRequest: express$Request) =>
+    new Promise<any>((resolve, reject) => {
```

Comment:
I'm not too familiar with flow generics, but that may be a way to avoid using the "any" type here.
https://flow.org/en/docs/types/generics/
```
get: function f<T>(...) { return new Promise<T>(...) }
```

Then later you can specify a type when using get
```
const x = get<string | null>(...);
if (x == null) { console.error("invalid response") }
else { console.log("X is " + x) }
```

Reply:
This code needs refactoring together with routes.js so I did not invest too much time in it yet. Generics seem to be a good solution if we keep the design, thanks.